CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Add `redactClasses` to Session Replay options ([#3546](https://github.com/getsentry/sentry-java/pull/3546))
+
```

Comment:
```suggestion
```
we can omit the changelog entry since we still haven't merged to main, so it will all be just `Features: Session Replay` eventually